### ISSUE TITLE
Pasting in input without updating data in Vuejs

### ISSUE THREAD

**Issue by JoohnCruz** [moip/awesome-mask#11]
I am having problems when the user paste the value of the field(ctrl + v). The field fullfil the mask correctly, but the element in the data property in vuejs stays without the mask. Have you been through this?

Code: 
```
import AwesomeMask from 'awesome-mask'

Vue.directive('mask', AwesomeMask);

new Vue ({
    el: '#inicialEmail',
    data: {
        action: '',
        requestType: '',
        inscricao: {
            nomcan: '',
            emailcan: '',
            celcan: ''
        }
    })
```

Example: 

Copy paste:

![image](https://cloud.githubusercontent.com/assets/7672428/23811175/9ca5c1e6-05b3-11e7-8a37-ebf7c1da314a.png)

Write:

![image](https://cloud.githubusercontent.com/assets/7672428/23811202/bd185e84-05b3-11e7-9734-bf84453b635e.png)

**Comment by iCaio** [comment]
Are you using which event on the template? Have you tried to get the value after `Vue.nextTick`?

**Comment by iCaio** [comment]
@JoohnCruz

**Comment by JoohnCruz** [comment]
@iCaio i will try to use the  `Vue.nextTick`, 

I just tried to use, important and as directive. 

Example: 

`import AwesomeMask from 'awesome-mask'

Vue.directive('mask', AwesomeMask);`

`<input name="cpfcan" id="cpfcan" v-model="inscricao.cpfcan" v-mask="'999.999.999-99'">`

No event is intended for this.

**Comment by iCaio** [comment]
After some hours looking for it, I found this:
https://github.com/vuejs/vue/issues/2804

This is related to this Vue issue.

I think I have fixed it on version 0.4.2, can you try it @JoohnCruz ?

**Comment by iCaio** [comment]
Issue closed due one week or ore of inactivity
